resource/readers/resource_reader_jgf.cpp

```diff
@@ -513,6 +535,30 @@ int resource_reader_jgf_t::add_graph_metadata (vtx_t v,
     return rc;
 }
 
+int resource_reader_jgf_t::update_vmap (std::map<std::string, 
+                                                 vmap_val_t> &vmap,
+                                        const vtx_t &v, 
```

Comment:
```suggestion
                                       vtx_t v, 
```

`vtx_t` is a native integer data type and this doesn't need to be `const` ref.